docs/src/tutorials/pino_ode.md

```diff
@@ -0,0 +1,66 @@
+# Physics Informed Neural Operator for ODEs Solvers
+
+This tutorial provides an example of how to use the Physics Informed Neural Operator (PINO) for solving a family of parametric ordinary differential equations (ODEs).
+
+## Operator Learning  for a family of parametric ODE.
+
+In this section, we will define a parametric ODE and solve it using a PINO. The PINO will be trained to learn the mapping from the parameters of the ODE to its solution.
+
+```@example pino
+using Test
+using OrdinaryDiffEq, OptimizationOptimisers
+using Lux
+using Statistics, Random
+using NeuralPDE
+
+equation = (u, p, t) -> cos(p * t)
+tspan = (0.0f0, 1.0f0)
+u0 = 1.0f0
+prob = ODEProblem(equation, u0, tspan)
+
+# Define the architecture of the neural network that will be used as the PINO.
+branch = Lux.Chain(
+    Lux.Dense(1, 10, Lux.tanh_fast),
+    Lux.Dense(10, 10, Lux.tanh_fast),
+    Lux.Dense(10, 10))
+trunk = Lux.Chain(
+    Lux.Dense(1, 10, Lux.tanh_fast),
+    Lux.Dense(10, 10, Lux.tanh_fast),
+    Lux.Dense(10, 10, Lux.tanh_fast))
+deeponet = NeuralPDE.DeepONet(branch, trunk; linear = nothing)
+
+bounds = (p = [0.1f0, pi],)
+db = (bounds.p[2] - bounds.p[1]) / 50
+dt = (tspan[2] - tspan[1]) / 40
+strategy = NeuralPDE.GridTraining([db, dt])
```

Comment:
Don't use grid training

Reply:
Ok, I will support also QuasiRandomTraining for PINO ODE and use it in Doc  example

Reply:
StochasticTraining*